airbyte-cdk/java/airbyte-cdk/db-destinations/src/main/kotlin/io/airbyte/cdk/integrations/destination/jdbc/typing_deduping/JdbcDestinationHandler.kt

```diff
@@ -392,7 +410,7 @@ abstract class JdbcDestinationHandler<DestinationState>(
                     {
                         map: LinkedHashMap<String?, String>,
                         column: Map.Entry<String?, ColumnDefinition> ->
-                        map[column.key] = column.value.type
+                        map[column.key] = column.value.type.lowercase()
```

Comment:
do we need to do this if we compare case-insensitively? (To be clear, both options are equivalent, and not 100% foolproof, but sufficient for now)

Reply:
this is unfortunately confusing. The ignore-case comparisons above only happen on the `_airbyte_*` columns. These comparisons are against all the non-airbyte columns. (the airbyte columns are removed in a `.filter` above)